pkgs/default.nix

```diff
@@ -169,6 +169,22 @@ in rec {
 
   jenkins = pkgs.callPackage ./jenkins {};
 
+  jmaps = stdenv.mkDerivation {
+    name = "jmaps";
+    src = fetchurl {
+      url = https://raw.githubusercontent.com/brendangregg/Misc/fc2856eb0d0d7fe7a3b3dfef8f873603f6a4a3eb/java/jmaps;
+      sha256 = "15bihx36qdjv25l1xdnkg3lrlmkjy9cmj6gsk4dybr141xfg9x4i";
+    };
+    unpackPhase = "true";
+    installPhase = ''
+      cp $src $out;
+      substituteInPlace $out --replace "/usr/lib/jvm/java-8-oracle" "${pkgs.openjdk}/lib/openjdk"
```

Comment:
Placement in patchPhase or something is fine, I'm just not sure about target paths with this style of `fetchurl`.